packages/clay-drop-down/src/DropDown.tsx

```diff
@@ -231,6 +238,14 @@ function ClayDropDown<T>({
 				'aria-controls': ariaControls,
 				'aria-expanded': internalActive,
 				'aria-haspopup': 'true',
+				children: (
+					<>
+						{trigger.props.children}{' '}
+						{triggerIcon !== 'undefined' && (
```

Comment:
When I referred to undefined, it would be the type and not the string. Sorry if I wasn't very clear with that.

```suggestion
						{triggerIcon && (
```